exercises/meetup/meetup_test.nim

```diff
@@ -0,0 +1,291 @@
+import unittest, times
+# import meetup
+import example
+
+# version 1.1.0
+
+suite "Meetup":
+  test "monteenth of May 2013":
+    check meetup(2013, 5, Teenth, dMon) == 13
```

Comment:
Looks like you are still working on this exercise, but are you planning on updating this to match what in the `canonical-data.json` for the `expected`, String of the Date or a Date?

Reply:
See the edited PR description. What do you think?

Sorry for the slow reply.